project_task_materials_analytic_partner/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Partner in task materials
+=========================
+
+This module adds the partner on task materials and propagate it to the
+auto-generated analytic entries on the "Other partner" field.
+
+
+
+Usage
+=====
+
+Go to Project > Tasks, and you will find the partner field in material lines.
```

Comment:
Add the text: `Using this partner, when you invoice these materials, the target of the invoice will be this partner instead the one in the project".